tests/test.js

```diff
@@ -1209,6 +1209,18 @@
                 same(get_props_without_distinct_id(mixpanel.test), props, "properties set properly");
             });
 
+            test('register_page_lifecycle', 3, function() {
```

Comment:
@tdumitrescu I'm not entirely sure of what the `3` means in the `test` function call. I'm also not sure how to test the _clear-upon-load_ behavior, do you mind sharing some info on that?